### DISCUSSION TITLE
How to have both a documentation and a blog?

### DISCUSSION THREAD

**Discussion by ftzi** [shuding/nextra#3788] [Q&A] [answered]
Title. From what I've read in the docs, we need to pick one of the two or the custom one. I want both a blog and a documentation.

Is it possible?

Thanks!

**Comment by ftzi** [comment, marked as answer]
Just saw https://github.com/shuding/nextra/issues/2571 and that it will available on v4. Waiting for it!

**Reply by huykon** [reply]
@ftzi so how can use both docs & blog in nextra v4?
